### PR TITLE
Detemplated CF books and added a simpler sol

### PR DESCRIPTION
**Pull request by TanishT**
the current internal sol has the benq template and was kinda confusing for me. Should I have replaced his sol with mine or should I leave them both there?

### CODE REVIEW

**Comment by bqi343** [comment]
This is not the same as the previous solution.

File: solutions/CF Books.mdx [hunk 1]
```diff
@@ -14,223 +14,41 @@ When moving the left pointer one unit to the right, the right pointer should be
...
-		res += "}"; return res;
-	#else
-		bool fst = 1; str res = "";
-		for (const auto& x: v) {
-			if (!fst) res += " ";
-			fst = 0; res += ts(x);
+int main(){
+	int n, t;	
+	cin >> n >> t;
+	int arr[n];
```
Comment (bqi343) [inline solutions/CF Books.mdx:31]: As noted [here](https://usaco.guide/bronze/intro-ds?lang=cpp), variable length arrays are not part of the C++ standard, so I think vector is preferred?

**Review by bqi343 — CHANGES_REQUESTED** [review]

**Comment by TanishT** [comment]
I added the previous solution back in and added my solution with a brief explanation at the bottom. Let me know if anything needs to be fixed.

File: solutions/CF Books.mdx [hunk 2]
```diff
@@ -14,6 +14,18 @@ When moving the left pointer one unit to the right, the right pointer should be
...
 
+Explanation of Sample Cases:
+1. Valera can read the books with times 1, 2, 1.
+2. Valera can only read one of the books. 
+
+<LanguageSection>
+
+<CPPSection>
+
+# C++ Implementation
```
Comment (bqi343) [inline solutions/CF Books.mdx:25]: > # C++ Implementation

remove this

File: solutions/CF Books.mdx [hunk 3]
```diff
@@ -14,6 +14,18 @@ When moving the left pointer one unit to the right, the right pointer should be
...
+
+## Solution 1
```
Comment (bqi343) [inline solutions/CF Books.mdx:27]: Only put code inside of LanguageSection.

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: solutions/CF Books.mdx [hunk 4]
```diff
@@ -17,211 +23,21 @@ than or equal to $t$. This way, we can find for each possible left end the
...
-		}
-		res += "}"; return res;
-	#else
-		bool fst = 1; str res = "";
-		for (const auto& x: v) {
-			if (!fst) res += " ";
-			fst = 0; res += ts(x);
-		}
-		return res;
+Our goal is to find the length of the longest subarray that has a sum of greater than or equal to $t$. In the first sample case, the subarray that has [$1,2,1$] sums to $5$ and has a length of $3$. The subarray [$3,1,2$] that sums to $6$ also works. In the second sample case, any of the subarrays with length $1$ will work.
```
Comment (bqi343) [inline solutions/CF Books.mdx:28]: > greater than or equal

less than or equal to?

> `[$1,2,1$]`

$[1,2,1]$ preferred

> The subarray that has [$1,2,1$] sums to $5$ and has a length of $3$.

1+2+1 is not 5?

> The subarray [$3,1,2$] that sums to $6$ also works.

I don't think so

Comment (bqi343) [reply solutions/CF Books.mdx:28]: $left$

File: solutions/CF Books.mdx [hunk 5]
```diff
@@ -17,211 +23,21 @@ than or equal to $t$. This way, we can find for each possible left end the
...
-	// throws exception when do smth illegal
-	// ex. try to read letter into int
-	if (sz(s)) setIn(s+".in"), setOut(s+".out"); // for USACO
-}
+
+<LanguageSection>
+
+<CPPSection>
+
+## Solution 1
```
Comment (bqi343) [inline solutions/CF Books.mdx:37]: Please put the implementation directly below the corresponding explanation (as in https://github.com/cpinitiative/usaco-guide/pull/1009/files)

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: solutions/CF Books.mdx (hunk 4 above)
Comment (bqi343) [reply solutions/CF Books.mdx:28]: shouldn't this go above both solutions?

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: solutions/CF Books.mdx (hunk 5 above)
Comment (bqi343) [reply solutions/CF Books.mdx:32]: ^ not resolved

**Review by bqi343 — CHANGES_REQUESTED** [review]
see unresolved comment above

File: solutions/CF Books.mdx [hunk 6]
```diff
@@ -2,28 +2,42 @@
...
 
 ```cpp
 #include <bits/stdc++.h>
 using namespace std;
-
+
```
Comment (bqi343) [inline solutions/CF Books.mdx:37]: also what's up w/ all these spaces :/

**Review by bqi343 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**